.gitignore

```diff
@@ -1,11 +1,12 @@
 tyk-oss/charts/*
-tyk-oss/Chart.lock
```

Comment:
just asking, why we deleted all lock files? Do they need to be tracked in git?

Reply:
Yes we need to. 
Also, we used to track them. We never `git rm` them, so they were never ignored.